_i18n/it/changes.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Tutte i cambiamenti recenti in QMK"
+layout: qmk-title
+permalink: /it/changes/
+---
+
+{% for change in site.changes %}
```

Comment:
This is a change that recently went into our main branch. It reverses the sort order of the Changes page so the newer updates are at the top.

```suggestion
{% for change in site.changes reversed %}
```